capabilities/software-development.html

```diff
@@ -0,0 +1,25 @@
+---
+layout: capability
+
+asset-name: software_development
+
+description: "Our software development team used the latest tools. See what we can build for you."
+
+tags: [Software Development]
+
+title: Software Development
+
+subtitle: Crafting solutions to automate your process
+
+text: 'Here at BG Software, we understand that there are many business needs that can be solved with Software.
+Through meeting with our clients and discussing their current workflow and processes, we help to design/craft
+the best possible solution, often
+<a class="color-teal hover-color-dark-blue" href="/case-studies/americasmart">automating</a> processes that were
+previously done manually and/or on paper. The software development process captures all the steps in between. Because
+the business is involved in requirements gathering and available for help in the design phase, the end product is unique
+and suits them. Software development doesn’t have to be a big project or application. It can be a small custom
+calculator on a website, adding new fields toan already existing submittal form, or implementing a single sign-on for
```

Comment:
Add a space after:
adding new fields to

Reply:
Yes

Reply:
My fault. Fixed.